frontend/README.md

```diff
@@ -0,0 +1,12 @@
+# Segmented CFMM Frontend
+
+A frontend application used to communicate with any Segmented CFMM contract.
+
+## How to run locally
+
+```bash
+cd frontend
+yarn # install dependencies
+yarn tailwind # generate required css
+yarn dev # run a development server
+```
```

Comment:
@pasqu4le This PR is ready for review. I decide to not support: `transfer`, `update_operators`, and `increase_observation_count` for this initial version, since their forms are a bit tricky to be done, and I don't want this to take longer. Let me know if you want me to include this in this version or not.

Also, this is how it is run locally, let me know if you have problem running this.

Reply:
instructions work just fine, but I think this should mention that temple (or others?) wallet extension needs to be installed first.

Reply:
I think it would be nice to mention in this page that the interface does not support all the entrypoints.

In particular that it does not support entrypoints that need to be called by a contract.

Reply:
@rinn7e I think you missed the second comment here

Reply:
Sorry missed this, I've added the limitation section and mention the Temple wallet as well. Currently, I only added the support for Temple wallet. Let me know if we need to support other wallet.